### PR TITLE
Add Windows instructions for getting Butane

### PR DESCRIPTION
**Pull request by Okeanos**
Contains instructions for standalone binary installation on Windows. I presume that the user installs GPG, e.g. via Git for Windows, on their own for signature verification and, hence, don't provide additional instructions for that.

I would've added instructions for e.g. [scoop](https://scoop.sh) as well but my pull request (lukesampson/scoop-extras/pull/6194) is currently in limbo 😞.

### CODE REVIEW

**Review by travier — APPROVED** [review]
LGTM. Thanks!

File: modules/ROOT/pages/producing-ign.adoc [hunk 1]
```diff
@@ -130,6 +130,23 @@ curl https://getfedora.org/static/fedora.gpg | gpg --import
 gpg --verify butane-x86_64-apple-darwin.asc
 ----
 
+==== Windows
```
Comment (jlebon) [inline modules/ROOT/pages/producing-ign.adoc:133]: Should this say e.g. `Windows (using PowerShell)`?

**Review by jlebon — APPROVED** [review]
Cool! One optional comment, but LGTM as is too.

File: modules/ROOT/pages/producing-ign.adoc (hunk 1 above)
Comment (Okeanos) [reply modules/ROOT/pages/producing-ign.adoc:133]: The Linux and macOS standalone procedures don't specify that this should be run in a shell and I assume the intended audience for Windows understands that as well. `cmd.exe` might be a contender and source of confusion, though, maybe? Hovering over the code snippet and the actual syntax should put these doubts to rest, though.

**Review by bgilbert — APPROVED** [review]
Thanks!

**Pull request merged**